hack/lib/golang.sh

```diff
@@ -158,11 +163,20 @@ readonly KUBE_TEST_PORTABLE=(
 # need to target server platforms.
 # These binaries will be distributed in the kubernetes-test tarball.
 # If you update this list, please also update build/release-tars/BUILD.
-readonly KUBE_TEST_SERVER_TARGETS=(
-  cmd/kubemark
-  vendor/github.com/onsi/ginkgo/ginkgo
-  test/e2e_node/e2e_node.test
-)
+kube::golang::server_test_targets() {
+  local targets=(
+    cmd/kubemark
+    vendor/github.com/onsi/ginkgo/ginkgo
+  )
+
+  if [[ "${OSTYPE:-}" == "linux"* ]]; then
```

Comment:
@ixdy , the `OSTYPE` works when `make quick-release`. 

Here's the diff to show `OSTYPE` when building
```
-        kube::log::status "Building go targets for ${platform}:" "${targets[@]}"
+        kube::log::status "Building go targets for ${platform}/${OSTYPE}:" "${targets[@]}"
```

And here's the output of `make quick-release`: 

```
MacPro:kubernetes klaus$ make quick-release
+++ [0402 08:34:10] Verifying Prerequisites....
+++ [0402 08:34:10] Using Docker for MacOS
+++ [0402 08:34:10] Building Docker image kube-build:build-6433d374d6-5-v1.7.5-2
+++ [0402 08:34:16] Syncing sources to container
+++ [0402 08:34:18] Running build command...
+++ [0402 08:34:25] Building the toolchain targets:
    k8s.io/kubernetes/hack/cmd/teststale
    k8s.io/kubernetes/vendor/github.com/jteeuwen/go-bindata/go-bindata
+++ [0402 08:34:25] Generating bindata:
    test/e2e/generated/gobindata_util.go
~ ~/test/e2e/generated
~/test/e2e/generated
+++ [0402 08:34:26] Building go targets for linux/amd64 (linux-gnu):
    cmd/kube-proxy
    cmd/kube-apiserver
    cmd/kube-controller-manager
    cmd/cloud-controller-manager
    cmd/kubelet
    cmd/kubeadm
    cmd/hyperkube
    vendor/k8s.io/kube-aggregator
    plugin/cmd/kube-scheduler
```